tests/http/auth-jwt-remote.yml

```diff
@@ -0,0 +1,89 @@
+config: !file tests/http/config/auth-jwt-remote.graphql
+name: Auth with JWT loaded from remote
+
+mock:
+  - request:
+      # TODO: http_spec mock cannot mock request to any client used by internal libraries
+      # so this mock doesn't work actually and added here only for reference
+      url: http://localhost:3000/__jwks
```

Comment:
IOs should be controlled by us. This would otherwise limit our ability to deploy on cloudflare/serverless/wasm envs.